src/client/src/containers/Details/index.tsx

```diff
@@ -28,24 +38,56 @@ interface IProps {
   intl: InjectedIntl;
 }
 
+type Props = IProps & IDispatchProps;
+
 const messages = defineMessages({
   none: {
     id: "details.none",
     defaultMessage: "None"
+  },
+  redirectLabel: {
+    id: "licenses.redirectLinkLabel",
+    defaultMessage: "{redirectLabel}"
+  },
+  genericRedirect: {
+    id: "redirectLabel.generic",
+    defaultMessage: "here"
   }
 });
 
 const Details = ({
   detailInfo,
   formatteDetailInfo,
   handleBackClick,
-  intl
-}: IProps) => {
-  const LinkRenderer = (props: any) => (
-    <a href={props.href} className={styles.link} onKeyUp={keyUpHandler}>
-      {props.children}
-    </a>
-  );
+  intl,
+  openRedirectModal
+}: Props) => {
+  const LinkRenderer = (props: any) => {
+    const genericRedirect: string = intl.formatMessage(
+      messages.genericRedirect
+    );
+    const redirectLinkLabel =
+      props.children[0].props.value === genericRedirect
+        ? props.href
+        : `${props.children[0].props.value} license link`;
```

Comment:
This was the best way I could think of to manage links that were done through 'React Markdown'. Since, the text is `here`, we cannot say the title where it is taking us. So the best bet was to mention the url of the site it is taking us to.

Reply:
That's smart! I think `${props.children[0].props.value} license link` needs to be internationalized though, with `formatMessage`

Reply:
This complete message constructed by redirectLinkLabel has been internationalized below before passing to the function.
`redirectLinkLabel: intl.formatMessage(messages.redirectLabel, {
              redirectLabel: redirectLinkLabel
            }),`